pkg/provider/resource_cluster.go

```diff
@@ -203,6 +205,41 @@ func (c *clusterResource) Schema(ctx context.Context, req resource.SchemaRequest
 					},
 				},
 			},
+			"maintenance_window": schema.SingleNestedBlock{
+				MarkdownDescription: "Custom maintenance window.",
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+				Attributes: map[string]schema.Attribute{
+					"is_enabled": schema.BoolAttribute{
+						MarkdownDescription: "Is maintenance window enabled.",
+						Optional:            true,
```

Comment:
```suggestion
						Optional:            true,
						Computed:         true,
```
Otherwise, terraform apply throws the following error when the `maintenance_window` block is not present in the tf file:
```
╷
│ Error: Provider produced inconsistent result after apply
│
│ When applying changes to biganimal_cluster.single_node_cluster, provider "provider[\"registry.terraform.io/enterprisedb/biganimal\"]" produced an unexpected new value: .maintenance_window: was absent, but now present.
│
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```
We need to add `Computed:true` to all the fields in the maintenance_window SingleNestedBlock.

Reply:
They're already there.